.gitignore

```diff
@@ -86,3 +86,8 @@ $RECYCLE.BIN/
 .vsconfig
 
 QuartersSDK/.vs/
+<<<<<<< HEAD
```

Comment:
@paulofer85 don't commit a merge conflict

Reply:
sorry for that there I pushed .gitignore without conflicts